src/Command/Core/InitCommand.php

```diff
@@ -58,19 +48,52 @@ protected function interact(InputInterface $input, OutputInterface $output)
         $styleHelper = $this->getHelper('gush_style');
         $adapters = $this->getAdapters();
 
-        if (null === $input->getOption('adapter')) {
-            $input->setOption(
-                'adapter',
-                $styleHelper->numberedChoice('Choose repository-manager', $adapters[0])
-            );
-        }
+        $input->setOption(
+            'repo-adapter',
+            $styleHelper->numberedChoice(
+                'Choose repository-manager',
+                $adapters[0],
+                GitHelper::undefinedToDefault($input->getOption('repo-adapter'))
+            )
+        );
 
-        if (null === $input->getOption('issue-tracker')) {
-            $input->setOption(
-                'issue-tracker',
-                $styleHelper->numberedChoice('Choose issue-tracker', $adapters[1])
-            );
-        }
+        $input->setOption(
+            'issue-adapter',
+            $styleHelper->numberedChoice(
+                'Choose issue-tracker',
+                $adapters[1],
+                GitHelper::undefinedToDefault($input->getOption('issue-adapter'))
+            )
+        );
+
+        $input->setOption(
+            'org',
+            $styleHelper->ask(
+                'Specify the repository organization name',
+                GitHelper::undefinedToDefault($input->getOption('org'))
+            )
+        );
+
+        $input->setOption(
+            'repo',
+            $styleHelper->ask('Specify the repository name', GitHelper::undefinedToDefault($input->getOption('repo')))
+        );
+
+        $input->setOption(
+            'issue-org',
+            $styleHelper->ask(
+                'Specify the issue-tracker organization name',
+                GitHelper::undefinedToDefault($input->getOption('issue-org'), $input->getOption('org'))
```

Comment:
i am not sure i follow the name of this static method, i guess what is its input and what is the output, why not then just pass the whole input? or rather a VO?